### ISSUE TITLE
`hypothesis` sort order

### ISSUE THREAD

**Issue by vincentarelbundock** [vincentarelbundock/marginaleffects#776]
Bug reported by @marcora in another issue.

These calls should produce the same results:

``` r
library(marginaleffects)

load(url("https://github.com/vincentarelbundock/modelarchive/raw/main/data-raw/gusto.rda"))

mod = glm(
  day30 ~ tx * sex + age,
  family = "binomial",
  data = gusto)

cmp = avg_comparisons(
  mod,
  type = "link",
  variables = list("tx" = "pairwise"),
  by = "sex"
) 

cmp |> hypotheses(hypothesis = "b4 - b3 = 0")
# 
#     Term Estimate Std. Error      z Pr(>|z|)  2.5 % 97.5 %
#  b4-b3=0  -0.0782      0.118 -0.663    0.507 -0.309  0.153
# 
# Columns: term, estimate, std.error, statistic, p.value, conf.low, conf.high

cmp$estimate[4] - cmp$estimate[3]
# [1] 0.04356261

avg_comparisons(
  mod,
  type = "link",
  variables = list("tx" = "pairwise"),
  by = "sex",
  hypothesis = "b4 - b3 = 0"
) 
# 
#     Term Estimate Std. Error      z Pr(>|z|)  2.5 % 97.5 %
#  b4-b3=0  -0.0782      0.118 -0.663    0.507 -0.309  0.153
# 
# Columns: term, estimate, std.error, statistic, p.value, conf.low, conf.high
```

**Comment by vincentarelbundock** [comment]
@marcora I think this should be fixed in version 0.11.2.9000 on Github.

Thanks for the report, and sorry for the delay.

**Comment by marcora** [comment]
Thanks Vincent! Keep up the good work. I am very grateful for all the effort you are putting in this fantastic package.
